.github/workflows/ci.yml

```diff
@@ -1,5 +1,11 @@
 name: CI
-on: [push]
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
```

Comment:
FWIW, look what we have in Station Desktop:

```yaml
on:
  pull_request:
  push:
    branches:
      - main
    tags:
      - '*'
```

https://github.com/filecoin-station/desktop/blob/fc7afa8edc2f3578cd1156635e1ad961ba2f727f/.github/workflows/ci.yml#L2C1-L8C12